### DISCUSSION TITLE
The default 5-fold cross-validation split

### DISCUSSION THREAD

**Discussion by tarash66** [MIC-DKFZ/nnUNet#1345] [Q&A]
Dear,

I would be so grateful if you could respond to my question. As I figured out, the default 5-fold cross-validation split works randomly on the dataset. Is it so? If yes, would you please inform me whether the split for the training of Task082_BraTS2020 was also random?

Regards,
Tarash

**Comment by FabianIsensee** [comment]
Splits are always random. How else would we get reliable results?

**Reply by tarash66** [reply]
Dear Fabian.

 I am so grateful for your response. I thought maybe you tried to keep a pattern like dividing the first 20 percent of the dataset for test and the rest for training in the first fold, then the second 20 percent of the dataset for testing in the 2nd fold, and so on and so forth until the 5th fold.  Unfortunately, I couldn't find any record of images in each split. I would appreciate it if you could inform me where I can see it.

Regards,

**Reply by FabianIsensee** [reply]
See splits_final.json in your preprocessed data folder

**Comment by tarash66** [comment]
I mean your "splits_final.json" for BraTS2020. I am intersted to know what images from BraTs2020 dataset were used for each provided model (out of 5). I appericate your response.

Regards,

**Reply by FabianIsensee** [reply]
Ah I see. 
[splits_final.zip](https://github.com/MIC-DKFZ/nnUNet/files/11077104/splits_final.zip)

Take this.
I had to zip it because GitHub doesn't allow pkl files...

**Reply by FabianIsensee** [reply]
All models are trained as 5 fold cross-validation with the splits from this file

**Reply by tarash66** [reply]
thanks dear Fabian. All the bests.
